syntax = "proto2";

package NetSceneQueryImgProto;


message NetSceneQueryImgReq {
  optional bytes img_bytes = 1;

}

message NetSceneQueryImgResp {
  enum ItemType {
    PLANT     = 0;
    ANIMAL    = 1;
    LANDMARK  = 2;
  }
  optional ItemType item_type = 1 [default = PLANT];
  optional string item_name = 2;
  optional string item_desc = 3;

}

